content/en/docs/OSSMC/users-guide.md

```diff
@@ -78,6 +78,19 @@ If you hover over a cell in a heatmap, a tooltip will give some details on the c
 
 ![Workload: Traces Heatmap](/images/documentation/installation/installation-guide/29-workload-traces-heatmap.png)
 
+When the OpenShift tracing UI plugin is enabled, Kiali will try to auto discover the plugin settings and the `View in Tracing` Kiali link will redirect to the plugin (from Kiali 2.8.0). 
```

Comment:
I just find showing the "+" makes it explicitly clear this was a feature introduced in 2.8.0 but is not limited to only 2.8 but is for all versions forward.

```suggestion
When the OpenShift tracing UI plugin is enabled, Kiali will try to auto discover the plugin settings and the `View in Tracing` Kiali link will redirect to the plugin (for Kiali 2.8.0+). 
```